src/query/api/v1/handler/prometheus/remote/write.go

```diff
@@ -93,6 +93,16 @@ var (
 		Attributes: ts.DefaultSeriesAttributes(),
 		Metadata:   ts.Metadata{},
 	}
+
+	headerToMetricType = map[string]prompb.MetricType{
+		"counter":         prompb.MetricType_COUNTER,
+		"gauge":           prompb.MetricType_GAUGE,
+		"gauge-histogram": prompb.MetricType_GAUGE_HISTOGRAM,
```

Comment:
nit: Perhaps `gauge_histogram` given `-` is not common in Prometheus naming conventions?